CODEOWNERS

```diff
@@ -36,10 +36,12 @@
 # Bootstrap and transit scripts
 /cmd/bootstrap/** @vishalchangrani
 
+# Automation Team
+/integration/** @gomisha
```

Comment:
can you please add the following directories under `Dev Tools Stream` and replace `Automation Team`:

`/insecure/**` @gomisha @yhassanzadeh13 
`/integration/benchnet2` @gomisha 
`/tools/test_monitor` @gomisha 
`.github/workflows` @gomisha 

Also, I think the `/integration/**` directory is too large to go all under me. Can we remove that and use sub-directories of `/integration/` to assign to different teams?